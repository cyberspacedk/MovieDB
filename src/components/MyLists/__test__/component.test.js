import React from 'react';
import { shallow } from 'enzyme';
import MyList from '../component';

describe('Component: MyList', () => {
  const props = {
    showModal: jest.fn(),
    hideModal: jest.fn(),
    visibleMod: false,
    deleteListRequest: jest.fn(),
    myLists: [{ id: 1 }],
    error: false,
    loading: false,
    empty: false,
    totalResults: 5,
    currentPage: 33,
    goToNextPage: jest.fn(),
    history: {
      push: jest.fn(),
    },
  };

  const container = shallow(<MyList {...props} />);

  it('Snapshot: should match', () => {
    expect(container).toMatchSnapshot();
  });
});
