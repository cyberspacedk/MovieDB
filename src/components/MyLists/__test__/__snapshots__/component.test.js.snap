// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: MyList Snapshot: should match 1`] = `
<Adapter>
  <Adapter>
    <Row
      gutter={0}
    >
      <Col
        offset={2}
        span={20}
      >
        <div
          className="top-margin"
        >
          <Title
            level={1}
          >
            My Lists
            <Icon
              onClick={[MockFunction]}
              type="plus-circle"
            />
          </Title>
        </div>
      </Col>
    </Row>
    <Row
      gutter={8}
      type="flex"
    >
      <Col
        offset={2}
        span={20}
      >
        <ListItemContainer
          deleteListRequest={[MockFunction]}
          history={
            Object {
              "push": [MockFunction],
            }
          }
          item={
            Object {
              "id": 1,
            }
          }
          key="1"
        />
      </Col>
    </Row>
    <Row
      gutter={0}
      justify="center"
      type="flex"
    >
      <Col>
        <Pagination
          className="pagination"
          current={33}
          defaultCurrent={1}
          defaultPageSize={20}
          onChange={[MockFunction]}
          total={5}
        />
      </Col>
    </Row>
  </Adapter>
  <Connect(WithFormik(CreateListFormModalContainer))
    hideModal={[MockFunction]}
    visibleMod={false}
  />
</Adapter>
`;
